ci/docker/image-builder/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM openjdk:11 as BUILD
+
+RUN mkdir /app
+COPY . /app
+WORKDIR /app
+
+# Skip integration tests because they use Docker.
+# This is impossible while building the image.
+# We can't mount /var/run/docker.sock.
+RUN ./gradlew build -x integrationTest
```

Comment:
I don't get this line.
I prefer to add a description to that file. What and how it builds

Reply:
This command runs a Gradle task to build the app.
This project is a standard Java CLI app.

> to that file

I don't get it. What file?

Reply:
I've added an explanation of working with the project to README.
This project is shipped in a container but before we work with it like with any other project.

https://github.com/avito-tech/avito-android/pull/1341/files#diff-46c7cb3f9fc9566533c7dd6923e8338086828b6f91a8ad244b96fe7090ba585c